managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedger.java

```diff
@@ -453,19 +453,23 @@
     /**
      * Update managed-ledger's properties.
      *
-     * @param properties key-values of properties
+     * @param properties  key-values of properties
+     * @param isOverwrite set true to overwrite original value of managed-ledger's properties; set false to append the
+     *                    properties to managed-ledger's properties.
      */
-    void setProperties(Map<String, String> properties) throws InterruptedException;
+    void setProperties(Map<String, String> properties, boolean isOverwrite) throws InterruptedException;
```

Comment:
Using `isOverwrite` feels a bit weird compared to similar properties APIs.

What about having: 
 * `setProperty()` 
 * `setProperties()` (with `isOverwrite==false` behavior)
 * `deleteProperty()`

Reply:
OK. Thanks for the suggestion, I will add these APIs.